homeassistant/components/lg_netcast/media_player.py

```diff
@@ -62,20 +67,22 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     host = config.get(CONF_HOST)
     access_token = config.get(CONF_ACCESS_TOKEN)
     name = config.get(CONF_NAME)
+    turn_on_action = config.get(CONF_ON_ACTION)
 
     client = LgNetCastClient(host, access_token)
 
-    add_entities([LgTVDevice(client, name)], True)
+    add_entities([LgTVDevice(client, name, hass, turn_on_action)], True)
```

Comment:
Please pass the script instance or `None` to the entity instead.

Reply:
I made a fixup so you can see the proposed changes separately. Lmk thx